src/mango/src/mango_cursor_view.erl

```diff
@@ -187,7 +206,10 @@ base_args(#cursor{index = Idx, selector = Selector, fields = Fields} = Cursor) -
             {selector, Selector},
             {callback_args, viewcbargs_new(Selector, Fields, undefined)},
 
-            {ignore_partition_query_limit, true}
+            {ignore_partition_query_limit, true},
+
+            % Request execution statistics in a map.
```

Comment:
At some point we could presumably remove this. Should we add a TODO or some kind of version marker to help make it easier to clean it up.  https://github.com/apache/couchdb/issues/4396